tailwind.config.js

```diff
@@ -0,0 +1,31 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
Remove unneeded imports

Reply:
HDF to HTML has been refactored to avoid needing tailwind on SAF CLI. Tailwind is all managed by hdf-converters's build system.